### DISCUSSION TITLE
badge

### DISCUSSION THREAD

**Discussion by aleebaba3594** [community/community#35033] [Other Feature Feedback, Questions, & Ideas] [answered]
How do I get the Quickdraw badge?

**Comment by DSDmark** [comment, marked as answer]
This is the simplest word how to get Quick draw badge

> (closed an issue / pull request within 5 minutes of opening)

Here is the list of all badge:-

> https://github.com/drknzz/GitHub-Achievements

**Reply by Carter-Mo** [reply]
How to close an issue?
